robot.js

```diff
@@ -96,16 +96,36 @@
 
     switch (robot.dir) {
       case 'up':
-        y = y < board.length - 1 ? y + 1 : y;
+        if (y < board.length - 1) {
+          y = y + 1;
+        } else {
+          y;
+        }  
+        //y = y < board.length - 1 ? y + 1 : y;
```

Comment:
When you do a pull request you are expected to take out all commented-out code. The reviewer is in general not interested in code that 'used to be'. Like you would send in a job application letter with paragraphs crossed out :grin: